docs/src/main/sphinx/release/release-367.md

```diff
@@ -0,0 +1,34 @@
+# Release 367 (XX Dec 2021)
+
+## General
+
+* Improve retryable query performance by adding support for dynamic filtering. ({issue}`10274`)
+* Improve `CREATE VIEW` error message when table or materialized view already exists. ({issue}`10186`)
+* Fix failure in `min_by` and `max_by` aggregation execution. ({issue}`10347`)
+* Fix planning failure when selecting all fields from a row type. ({issue}`10321`)
+* Fix potential hang when processing broadcast joins. ({issue}`10344`)
+* Fix failure when row or array in `VALUES` cause contains nulls. ({issue}`10141`)
```

Comment:
This does not read correctly.
@martint you have context. Can you fix phrasing.

Reply:
Yes, the word "cause" shouldn't be there.

Reply:
It should be "clause"